amplify/data/resource.ts

```diff
@@ -15,8 +17,16 @@ const schema = a
         email: a.string(),
         firstName: a.string(),
         lastName: a.string(),
+        totalEarnings: a.float(),
+        userId: a.id(),
+        sessionsAttended: a.belongsTo("SessionAttended", "userId"),
```

Comment:
A user will have many sessions attended, not belong to a session itself. Right now the logic is not correct because a user can only have 1 session, meaning that we would not be able to have logic to track their overall profit.